ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -315,26 +317,28 @@ cp "${INSTALL_DIR}/auth/kubeadmin-password" "${SHARED_DIR}/"
 ### Mount the image in the hosts via virtual media
 ### We cannot mount the ISO image before it gets created. Therefore, despite the increased complexity of this step,
 ### we need to do it here
-echo -e "\nMounting the ISO image in the hosts via virtual media and powering on the hosts..."
 # shellcheck disable=SC2154
-for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
-  # shellcheck disable=SC1090
-  . <(echo "$bmhost" | yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")')
-  if [ "${transfer_protocol_type}" == "cifs" ]; then
-    IP_ADDRESS="$(dig +short "${AUX_HOST}")"
-    iso_path="${IP_ADDRESS}/isos/${CLUSTER_NAME}.${arch}.iso"
-  else
-    # Assuming HTTP or HTTPS
-    iso_path="${transfer_protocol_type}://${AUX_HOST}/${CLUSTER_NAME}.${arch}.iso"
-  fi
-  mount_virtual_media "${bmc_address}" "${redfish_user}" "${redfish_password}" \
-   "${iso_path}" "${transfer_protocol_type}" &
-done
+if [ "${BOOT_MODE}" == "iso" ]; then
```

Comment:
We can move this after line 290 this block (without the `if`).

Reply:
Done